peakinvestigator/lib/__init__.py

```diff
@@ -0,0 +1,20 @@
+## -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+import os
+import io
+import logging
+import socket
+import ssl
+import struct
+import time
+import tarfile
+import zipfile
+from tqdm import tqdm
```

Comment:
I don't think all of these system modules need to be imported here. If there are errors without, they probably need to be imported in the individual progress module.